packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.Props.ts

```diff
@@ -72,4 +72,9 @@ export interface IBreadcrumbItem {
    * If this breadcrumb item is the item the user is currently on, if set to true, aria-current="page" will be applied to this breadcrumb link
    */
   isCurrentItem?: boolean;
+
+  /**
+   * Switch to toggle whether or not breadcrumb item should be focusable. Default is true.
+   */
+  dataIsFocusable?: boolean;
```

Comment:
I don't think we want to be doing this.

By default, buttons and anchors are focusable. So this really should be unnecessary.